src/javascripts/legacy_app.js

```diff
@@ -1,8 +1,8 @@
 import BaseApp from 'base_app';
 
-// remove these if not needed
```

Comment:
Maybe I can reword the comment to say "uncomment these if needed or remove otherwise"?